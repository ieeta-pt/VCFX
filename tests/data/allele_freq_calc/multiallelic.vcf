##fileformat=VCFv4.2
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SAMPLE1	SAMPLE2	SAMPLE3
1	100	rs1	A	G,T	30	PASS	AF=0.25,0.10	GT	0/1	0/2	1/2
1	200	rs2	C	T,G,A	40	PASS	AF=0.5,0.1,0.2	GT	0/1	2/3	1/2
