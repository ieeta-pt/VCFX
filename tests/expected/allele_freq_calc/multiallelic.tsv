CHROM	POS	ID	REF	ALT	Allele_Frequency
1	100	rs1	A	G,T	0.6667
1	200	rs2	C	T,G,A	0.8333
